### PR TITLE
CLC-5216, get_all_course_sections added to queries.rb; plus query_spec upgrade

### PR DESCRIPTION
**Pull request by johncrossman**
https://jira.ets.berkeley.edu/jira/browse/CLC-5216

Bamboo: https://bamboo.ets.berkeley.edu/bamboo/browse/MYB-CJT50-4

mediacasts_controller will invoke the query get_all_course_sections to pull ccn list per dept, catalog_id. The ccn list will be passed to Webcast code (future PR) and return zero or more sets of recordings.

### CODE REVIEW

**Comment by johncrossman** [comment]
Travis good on one of two

File: app/models/campus_oracle/queries.rb [hunk 1]
```diff
@@ -307,22 +307,11 @@ def self.get_cross_listings(term_yr, term_cd, ccns)
...
-        c.instruction_format, c.section_num
-        SQL
-        result = connection.select_all(sql)
-      }
-      stringify_ints! result
+      stringify_ints! get_course_sections(term_yr, term_cd, department, catalog_id, true)
+    end
+
+    def self.get_all_course_sections(term_yr, term_cd, department, catalog_id)
+      stringify_ints! get_course_sections(term_yr, term_cd, department, catalog_id, false)
```
Comment (paulkerschen) [inline app/models/campus_oracle/queries.rb]: @johncrossman : looks like either these public methods or the private method can lose `stringify_ints!`

File: spec/models/campus_oracle/queries_spec.rb [hunk 2]
```diff
@@ -426,20 +430,20 @@
...
-    it "returns false if string is not successfully converted to an integer" do
-      expect(CampusOracle::Queries.is_integer_string?("18dfsd9023")).to be_falsey
-      expect(CampusOracle::Queries.is_integer_string?("254AbCdE")).to be_falsey
-      expect(CampusOracle::Queries.is_integer_string?("98,()@")).to be_falsey
-      expect(CampusOracle::Queries.is_integer_string?("2390.023")).to be_falsey
+    it 'returns false if string is not successfully converted to an integer' do
+      expect(CampusOracle::Queries.is_integer_string?('18dfsd9023')).to be false
+      expect(CampusOracle::Queries.is_integer_string?('254AbCdE')).to be false
+      expect(CampusOracle::Queries.is_integer_string?('98,()@')).to be false
+      expect(CampusOracle::Queries.is_integer_string?('2390.023')).to be false
```
Comment (paulkerschen) [inline spec/models/campus_oracle/queries_spec.rb]: @johncrossman thank you for the massive syntax cleanup!

**Comment by johncrossman** [comment]
stringify_ints! dropped.  Good catch @paulkerschen

**Comment by johncrossman** [comment]
Once again, 50% goodness on Travis

**Pull request merged**